pkg/cmd/server/origin/master.go

```diff
@@ -305,14 +296,30 @@ func (c *MasterConfig) InstallAPI(container *restful.Container) []string {
 	}
 
 	var root *restful.WebService
+	installed := 0
```

Comment:
There's a lot of installing happening in this method.  How about something a little more specific userFiltersApplied or maybe, 'theNumberOfTimesIAppliedTheCurrentUserContextFilterToARouteInMyServerForSecurityReasons'?